libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -55,6 +55,10 @@ enum CmRequest {
     ListConnections {
         sender: Sender<Result<Vec<String>, ConnectionManagerError>>,
     },
+    AddInboundConnection {
```

Comment:
I've kept the name, since it is still requesting a connection via an endpoint string (request makes sense as it could fail in ways that add_inbound connection cannot).  I have renamed the message variant for to `RequestOutboundConnection`